client/CServerHandler.h

```diff
@@ -25,13 +25,12 @@ class CGameState;
 struct ClientPlayer;
 struct CPack;
 struct CPackForLobby;
+struct SharedMemory;
```

Comment:
SharedMemory struct is defined in vcmi/interprocess.h inside lib namespace, so adding this outside of namespace  is not valid.
Instead, you can remove forward declaration SharedMemory from CServerHandler as it's unused

Reply:
> SharedMemory struct is defined in vcmi/interprocess.h inside lib namespace, so adding this outside of namespace is not valid.

Yes, this is why I moved it under lib namespace.

>  Instead, you can remove forward declaration SharedMemory from CServerHandler as it's unused as I can see

Right, makes sense! Maybe it compiled because it wasn't used. But for some reason failed on my compiler.
Removed.